project.clj

```diff
@@ -47,7 +44,7 @@
                                   [puppetlabs/trapperkeeper]
                                   [puppetlabs/trapperkeeper :classifier "test" :scope "test"]
                                   [puppetlabs/kitchensink :classifier "test" :scope "test"]
-                                  [puppetlabs/trapperkeeper-webserver-jetty9]]}
+                                  [puppetlabs/trapperkeeper-webserver-jetty9 "2.0.1-SNAPSHOT"]]}
```

Comment:
I don't see a PR for trapperkeeper-webserver-jetty9. So what changes are expected there?